grain/_src/python/data_loader_test.py

```diff
@@ -165,7 +165,15 @@ class DataLoaderTest(absl_parameterized.TestCase):
 
   def setUp(self):
     super().setUp()
-    self.testdata_dir = pathlib.Path(FLAGS.test_srcdir) / "testdata"
+
+    import grain
+    if grain.__file__ is None:
+        # tests run under bazel
+        self.testdata_dir = pathlib.Path(FLAGS.test_srcdir) / "testdata"
```

Comment:
should we try setting this flag in the pytest scenario somehow?

Reply:
pytest seems to have a fixture api through a conftest file that allows to provide initialization functions, e.g. in https://docs.pytest.org/en/stable/reference/reference.html#pytest.hookspec.pytest_sessionstart

the reason why I'm suggesting this is that the conftest.py file would be shared across all tests + would only be relevant to pytest automatically. As is, we'd have to update every test accessing local file system

Reply:
Currently the updated PR uses module-level fixtures in the modules which actually need this. We can make it into a session-level fixture, if you are not happy with module-level fixtures?